lib/elixir/lib/calendar/iso.ex

```diff
@@ -11,6 +11,8 @@ defmodule Calendar.ISO do
 
   @behaviour Calendar
 
+  @gregorian_epoch 1
```

Comment:
Implementation using `:calendar` module is simple; only subracting 365 days to convert between Rata Die and the `gregorian_days` format that `:calendar` uses is necessary.

However, `:calendar` will not work with dates before the 'year 0' so I am now in dubio what I should do. See discussion below.